docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,643 @@
+        <IconButton icon="ellipsis" iconColor="darkGray" size="lg" accessibilityLabel=""/>
+      </Tooltip>
+    }
+  />
+  <Box
+    display="flex"
+    wrap
+    width="80%"
+    direction="column"
+    marginStart={5}
+  >
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Heading size="sm" accessibilityLevel={2}>Edit profile</Heading>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <TextField
+        label="TextField 4"
+        id="textfield4"
+        onChange={() => {}}
+        placeholder="Placeholder"
+      />
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Box display="flex" wrap marginStart={-3} marginEnd={-3} marginBottom={-3} marginTop={-3}>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 5"
+            id="textfield5"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 6"
+            id="textfield6"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+      </Box>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <SelectList
+        label="SelectList"
+        id="selectlist0"
+        options={[
+          {
+            value: 'belgium',
+            label: 'Belgium',
+          },
+          {
+            value: 'france',
+            label: 'France',
+          },
+          {
+            value: 'usa',
+            label: 'USA',
+          },
+        ]}
+        placeholder="Placeholder"
+        onChange={() => {}}
+      />
+    </Box>
+  </Box>
+</Flex>;
+        `}
+        shaded
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`Be sure to follow any accessibility guidelines for the components used within PageHeader. The heading within PageHeader will be rendered as a level 1 heading, so ensure there are no other level 1 headings present on the page. `}
+  />,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`title\`, \`subtext\` and actions within PageHeader.`}
+  >
+    <MainSection.Card
+      shaded
+      cardSize="lg"
+      defaultCode={`
+<PageHeader
+  title="Langer Seitentitel, der in einigen Sprachen möglicherweise abgeschnitten wird"
+  subtext="2.131 Produkte"
+  primaryAction={<Button color="red" size="lg" text="Schnell erstellen" />}
+/>;
+  `}
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      description={`PageHeader supports an optional \`primaryAction\`. It can be a [Button](Button), a [Link](/Link) or an [IconButton](/IconButton) with a [Tooltip](/Tooltip) and optional [Dropdown](/Dropdown). Any Buttons or IconButtons should be \`size="lg"\`.`}
```

Comment:
Are we missing the IconButton with no Tooltip variant? 
![image](https://user-images.githubusercontent.com/10593890/111221565-0be20700-85b1-11eb-8acb-52fede09d79e.png)

Reply:
nit: no article "a"